src/health/views.py

```diff
@@ -12,7 +12,6 @@
 except ImportError:
     from django.db.models.loading import get_model
 
-from django.http import HttpResponse
```

Comment:
Make sure the health view still works. There might be a reason why it was down here.

Reply:
Works, thanks.